data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1611 @@
+	on waypoint
+		dialog "You have patrolled all six systems."
+	on complete
+		payment 250000
+		conversation
+			`Back on Ssil Vida, you check in with Gavriil and report the destruction of three Korath chasers that you located. "Three?" He signs quizzically. "That is an odd number. Raiders only carry two at most. I will report this right away. But in the meantime, come take a look at what I have set up." He leads you off towards the spaceport.`
+
+
+mission "Remnant: Cognizance 26"
+	name "Bring an Aeon Cell to Ssil Vida"
+	description "Gavriil has asked you to go pick up an Aeon Cell from Viminal by <date>."
+	deadline 14
+	source "Ssil Vida"
+	stopover "Viminal"
+	to offer
+		has "Remnant: Cognizance 25: done"
+	on offer
+		conversation
+			`Gavriil strides into the room with confidence. "look at this!" He gestures at the smooth sheeting covering the walls, and the new equipment quietly humming away off to the side. A large panel with a diagram of the station stands out in the middle of it.`
+			scene "scene/ssil vida alert hologram"
+			`	"Setting up the life support systems as self-contained environment within the station was simple enough that I did some preliminary power tracing on the station itself. It seems that the station is currently being powered by a immense fusion reactor that is clearly reaching its end of life." He pauses, then continues. "Not to mention it is basically running on the dregs of the dregs of the remains of its spent fuel. I do not know how much longer it will last, but likely no more than a few weeks. As best I can tell, it is still trying to power everything, but the main load," He points to the glowing red sphere on the hologram. "Seems to have simply shutdown once the power levels dropped below a certain threshold." He frowns at the display.`
+			`	"I do not know what will happen to the station when the main reactor fails completely, and we are not interested in letting this station collapse before we have had the chance to thoroughly investigate it. As such, we need to take action now to give ourselves room to work." He turns towards you. "I have contacted Viminal, and they have an Aeon Cell ready for transport. Please go pick it up and transport it back here." He smiles faintly. "Oh, and it does not matter whether you transport it installed on your ship or in your cargo, but make sure it is not your primary power source: I have the tools to uninstall it from your ship and set it up to connect to the station, but I do not have the equipment to install it back into your ship." He pauses to check a readout on the display. "Remember: It is vital that I have the Aeon Cell by <date>.`
+				accept
+	on stopover "Viminal"
+		outfit "Aeon Cell" 1
+		dialog "The Remnant have an Aeon cell ready when you land. They carefully move it to your ship, and several touch it reverently before they depart. You remind yourself to make sure that it is suitably stowed before taking off again."
```

Comment:
Out of curiosity, nothing happens if you fail, right? Would be fun if you fail and the entire station just disappears.

Reply:
Yes, it would be amusing. It's one of the more dramatic "what happens to the story if the player fails and things go differently" moments I've got marked down for development someday. I do want to have a number of divergent points in the story to make things more interesting, but my priority is having a single story complete first.